promql/test.go

```diff
@@ -51,6 +52,15 @@ const (
 
 var testStartTime = time.Unix(0, 0).UTC()
 
+//go:embed testdata
+var TestManifestFS embed.FS // TestManifestFS contains a set of tests.
+
+// TestEngine is the interface for an engine under test.
+type TestEngine interface {
```

Comment:
```suggestion
type EngineQuerier interface {
```

we are in test already

Reply:
:heavy_check_mark: